angular-src/src/app/components/adv-search/adv-search.component.html

```diff
@@ -1,49 +1,76 @@
+      </mat-form-field>
+      <mat-form-field *ngIf="selectedCategory?.type === 'number'" >
+        <input matInput [placeholder]="selectedCategory.name" type="number" ngModel name="categoryValue" class="category-value">
+      </mat-form-field>
+      <mat-select *ngIf="selectedCategory" placeholder="Order" name="order" ngModel class="search-order">
+        <mat-option value="ASC">Ascending</mat-option>
+        <mat-option value="DESC">Descending</mat-option>
+      </mat-select>
+      <div>
+        <button mat-button matStepperPrevious (click)='reset(searchForm.value)'>Back</button>
+        <button mat-button matStepperNext>Next</button>
+      </div>
+    </mat-step>
+    <mat-step [stepControl]='thirdFormGroup'>
+      <ng-template matStepLabel>Return The Following Data:</ng-template>
+      <div class="row">
+        <div class="col-sm-3 text">
+          <div class="card card-outline-success mb-3">
+            <div class="card-header card-inverse card-success">
+              Drag Selected Items to Drop Box
             </div>
-            <div class="return-check-container">
-                <mat-checkbox [disabled]="searchForm?.controls.returnType?.value === 'cards'" [(ngModel)]="requestedColumns.adult" name="adult">Is Adult?</mat-checkbox>
-            </div>
-            <div class="return-check-container">
-                <mat-checkbox [disabled]="searchForm?.controls.returnType?.value === 'cards'" [(ngModel)]="requestedColumns.movie_id" name="movie_id">Movie ID#</mat-checkbox>
+            <br>
+            <div class="card-block scroll-list">
+              <div class="list-group">
+                <li class="list-group-item list-group-item-action list-group-item-success" [draggable] *ngFor="let column of searchCategory"
+                  [dragClass]="'active'" [dragData]="column"> {{column.name}}
+                </li>
+              </div>
             </div>
+          </div>
         </div>
-        <div class="submit-row">
-            <button mat-fab color="primary" type="submit">Go!</button>
+        <div class="col-sm-3">
+            <div class="card card-outline-primary mb-3" droppable [dragHintClass]="'drag-hint'" (onDrop)="onItemDrop($event)">
+              <div class="card-header card-inverse card-primary">Drop Selected Items to Return</div>
+              <br>
+              <div class="card-block scroll-list">
+                  <li *ngFor="let column of droppedItems"
+                  [(ngModel)]="requestedColumns[column.ref]"
+                  [name]="column.ref" ngDefaultControl class="list-group-item">{{column.name}}</li>
+              </div>
+            </div>
         </div>
-    </form>
+        <div class="col-sm-6"></div>
+      </div>
 
-    <!-- <button (click)="whoAmI()">Who am i</button> -->
+      <div class="submit-row">
+        <button mat-button matStepperSubmit type="submit">See Results!</button>
```

Comment:
TODO: Eventually this should prevent submission if skip steps if it makes sense to do so.

Reply:
Sounds good.  I was running into way too much trouble getting the stepper to require each step however, we should fix this soon.